addons/modules/stringtable.xml

```diff
@@ -1724,6 +1724,12 @@
             <Polish>Dodaj wyposażenie</Polish>
             <Russian>Добавить снаряжение</Russian>
         </Key>
+        <Key ID="STR_ZEN_Modules_RemoveNVG">
+            <English>Remove NVGs</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_RemoveNVG_Tooltip">
+            <English>Remove any Night Vision Goggles currently equiped.</English>
```

Comment:
I think: "Remove any currently equipped Night Vision Goggles." reads a bit better. Also, equipped is misspelled.